lib/src/maintenance.dart

```diff
@@ -552,3 +555,25 @@ Future<Maintenance> detectMaintenance(
 
 bool _isMarkdown(String fileName) =>
     p.extension(fileName).toLowerCase() == '.md';
+
+bool _hasExampleInReadmeMarkdown(String text) {
+  if (text == null || text.isEmpty) return false;
+  final lines = text.split('\n').where((l) => l.trim().isNotEmpty).toList();
```

Comment:
What about:
```md
Examples
---------
....
```

this is also valid markdown...


I would propose:
1) not doing this
2) adding an `extractTableOfContents` to the `markdown` package on pub (we don't own it, but they'll probably accept contributions).

Reply:
Good point.